### ISSUE TITLE
Impossible to show iron atoms as spheres.

### ISSUE THREAD

**Issue by kovalp** [3dmol/3Dmol.js#632]
**Describe the bug**
The viewer stops working when iron atoms are in the system and spheres spec has 'scale' property. 

**To Reproduce**

1. Download an sdf file with iron atom. https://pubchem.ncbi.nlm.nih.gov/compound/Iron_III_-oxide#section=2D-Structure

2. Try to show the structure
```
viewer = $3Dmol.createViewer('div', {})
viewer.setStyle({sphere: {scale: 0.2, radius: 0.2}})
viewer.addModel(<content of the iron-containing sdf file>)
viewer.render()
```

3. The error in the debugger
![image](https://user-images.githubusercontent.com/8431900/206913802-b885ab71-3edc-44ee-8491-fd4008b0024f.png)

**Expected behavior**
I expect to see the model composed of spheres.

**Screenshots**
The viewer renders empty.

**Desktop (please complete the following information):**
 - OS: [Debian 11]
 - Browser [firefox]
 - Version [102.5.0esr]

**Additional context**
The structure could be shown using sticks style.

**Comment by dkoes** [comment]
Can you provide the actual file you are using that causes the problem?  When I use the 2D structure from PubChem it works fine (tested on Chrome and Firefox on MacOS).  Does this link cause the error?

https://3dmol.org/viewer.html?url=https%3A%2F%2Fpubchem.ncbi.nlm.nih.gov%2Frest%2Fpug%2Fcompound%2FCID%2F518696%2Frecord%2FSDF%2F%3Frecord_type%3D2d%26response_type%3Ddisplay&select=all&style=sphere

**Comment by dkoes** [comment]
This is a bit of a shot in the dark - do you by any chance have some developer extension installed that enforces type constraints (which would normally be ignored in Javascript)?

**Comment by kovalp** [comment]
Hi,

Yes, this link causes the trouble after adding attribute 'scale'.


Peter

**Comment by kovalp** [comment]
No, I don't think I have such extensions.

**Comment by kovalp** [comment]
Here is one file

[Structure2D_CID_518696.zip](https://github.com/3dmol/3Dmol.js/files/10220377/Structure2D_CID_518696.zip)

I think maybe there is no vdW radius specified for iron?

**Comment by dkoes** [comment]
Sorry to take so long to get back - the code was in the middle of a major refactor that had to be finished.  Fortunately, converting the code to TypeScript automatically revealed the source of this bug and it is now fixed in 2.0:
https://3dmol.org/tests/auto/generate_test.cgi?test=testfescale

**Comment by kovalp** [comment]
Cool!
